tests/publiccloud/check_registercloudguest.pm

```diff
@@ -31,9 +32,53 @@ sub run {
 
     select_host_console();    # select console on the host, not the PC instance
 
+    registercloudguest($instance) if is_chost();
+
+    if (is_byos()) {
+        # We need to skip all BYOS checks for CHOST BYOS images because of custom registration we doing above.
```

Comment:
Please also note that CHOST images come only as BYOS, e.g.

```suggestion
        # Container-Host images (CHOST) come only as BYOS - Here We skip all checks because of a custom registration procedure (see above)
```

Reply:
this is very important point which will change NOT only comment . thanks Felix :+1: